concurrency/MpscQueue.h

```diff
@@ -55,9 +56,9 @@ class MpscQueueBase
 #endif
         std::atomic<State> state;
         uint32_t size;
-        uint32_t _align32;
         uint8_t data[];
     };
+    static_assert(sizeof(Entry) % alignof(uint64_t) == 0, "Entry must allow 64bit alignment of data");
```

Comment:
If you follow my previous suggestion you can add this static_assert:
`static_assert(alignof(Entry) == alignof(uint64_t))`;

Reply:
Actually I am a bit surprised how how that works as the struct requires 32bit alignment. I was expecting the size to be 12 in release 🤔

Reply:
I make some experiments, interesting that `data[]`is considered an incomplete type and seems the sizeof Entry is 8. But the requirement alignment is still 32bits. I wondering if `uint8_t data[]` is actually the best data type to use in this case. Perhaps a `void*` could be better and we know it would be 64bit aligned